lib/govuk_app_config/govuk_error.rb

```diff
@@ -11,9 +12,11 @@ def self.notify(exception_or_message, args = {})
     Raven.capture_exception(exception_or_message, args)
   end
 
+  def self.init(raven_configuration = DefaultConfiguration.new(Raven.configuration))
```

Comment:
I don't think this needs an argument - as I don't think we'd expect anyone to alter it. It'd feel more idiomatic defined within the method, as otherwise it's a slightly confused setter.

Reply:
Thanks - I initially intended for it to be easily swapped in for testing purposes, in an Open for Extension/Closed for Modification principle. But as I went on, the tests changed enough that it wasn't strictly needed anymore.

I seem to remember attempting to move it into the method itself, but that caused one failing test (in `configuring_spec.rb`). I'll have another go though 👍